Microsoft.Research/Contracts/System.ComponentModel.Composition/Sources/System.ComponentModel.Composition.Hosting.ExportProvider.cs

```diff
@@ -67,12 +67,16 @@ public Lazy<T> GetExport<T>(string contractName)
 
     public T GetExportedValue<T>(string contractName)
     {
+      Contract.Ensures(Contract.Result<T>() != null); 
+      // because of ImportCardinality.ExactlyOne =>
```

Comment:
:bulb: This might be clearer if the comment was placed before the call to `Contract.Ensures`.